Makefile

```diff
@@ -14,12 +14,19 @@ up-passkey:
 	&& docker-compose -f docker-compose.yml -f docker-compose.passkey.yml up \
 	--no-recreate --detach ${LIMIT}
 
+up-keycloak:
+	docker-compose -f docker-compose.yml -f docker-compose.keycloak.yml up \
+	--no-recreate --detach ${LIMIT}
+
 stop:
 	docker-compose stop
 
 down:
 	docker-compose down
 
+down-keycloak:
+	docker-compose -f docker-compose.yml -f docker-compose.keycloak.yml down
```

Comment:
Do we really need another target to bring it down? I guess it's because you are adding a new container but I want to make sure

Reply:
What if I modify the down target to down using all the docker-compose files?

Reply:
That's fine for me, but does it work correctly if we aren't using any override?

Reply:
Seems to:

I first ran make up (no overrides just default docker-compose.yml.   This is all that was running:

```
# podman ps
CONTAINER ID  IMAGE                          COMMAND     CREATED        STATUS        PORTS       NAMES
2c9465cd0f1e  quay.io/sssd/ci-samba:latest   /sbin/init  3 minutes ago  Up 3 minutes              samba
effe4d655585  quay.io/sssd/ci-client:latest  /sbin/init  3 minutes ago  Up 3 minutes              client
07ba4196ab25  quay.io/sssd/ci-dns:latest                 3 minutes ago  Up 3 minutes              dns
771946867f3c  quay.io/sssd/ci-ldap:latest    /sbin/init  3 minutes ago  Up 3 minutes              ldap
e5cc9596b515  quay.io/sssd/ci-ipa:latest     /sbin/init  3 minutes ago  Up 3 minutes              ipa
57ebc7dfde51  quay.io/sssd/ci-kdc:latest     /sbin/init  3 minutes ago  Up 3 minutes              kdc
2f82dfb1cd88  quay.io/sssd/ci-nfs:latest     /sbin/init  3 minutes ago  Up 3 minutes              nfs
```

Then I ran make down using all 3 files:

```
# make down
docker-compose -f docker-compose.yml \
-f docker-compose.keycloak.yml \
-f docker-compose.passkey.yml down
WARNING: The HIDRAW variable is not set. Defaulting to a blank string.
Stopping dns    ... done
Stopping samba  ... done
Stopping nfs    ... done
Stopping kdc    ... done
Stopping ldap   ... done
Stopping ipa    ... done
Stopping client ... done
Removing dns    ... done
Removing samba  ... done
Removing nfs    ... done
Removing kdc    ... done
Removing ldap   ... done
Removing ipa    ... done
Removing client ... done
Removing network sssd-ci
```

On RHEL8 with podman-compose from pip, it does throw some errors but exits with rc 0.   And it does that for any/all containers that are not up not just the additional ones from the override files.